src/templates/doe_workflow_attic_insulation.mdx

```diff
@@ -0,0 +1,188 @@
+<Tabs>
+    <Tab eventKey="project" title="Project">
+        The data for this tab should be know prior to the initial site visit.
+
+        <ProjectInfoInputs {...props} />   
+
+    </Tab>
+    <Tab eventKey="installation" title="Installation">
+        
+        <DateInput label="Installation Date" path="install_date" />
+        
+        ## Building
+        <PhotoInput id="building_number_photo" label="Building Number – Photo">
+         Provide a photo of the building that shows the street number.
+        </PhotoInput>
+
+        Skip photo if feature is N/A to house.
+        
+        <PhotoInput id="insulation_measuring_rulers " label="Insulation Measuring Rulers  – Photo ">
+            Overview of insulation measuring rulers, one installed every 300 sq. ft. visible 
+            from attic entrance (photo taken from attic entrance).     
+        </PhotoInput>
+
+        <PhotoInput id="rvalue" label="R-value Per Inch  – Photo ">
```

Comment:
Let's make the id for this more descriptive. Maybe "rvalue_table_photo".

Reply:
done